### ISSUE TITLE
Basic alternative input

### ISSUE THREAD

**Issue by samreid** [phetsims/center-and-spread#10]
My understanding is that new sims would support basic keyboard navigation.  I think we should enable that for this sim.  @amanda-phet sound good?

**Comment by samreid** [comment]
@kathy-phet agreed we can add basic support for checkboxes as we go. But maybe wait until later to add soccer ball dragging.  But @kathy-phet also recommended checking on this at dev meeting to make sure it's a good plan.

**Comment by pixelzoom** [comment]
For fourier-making-waves, I added alt input all at once, when the sim was (near?) feature-complete.  But that was because the alt input requirement was added late in the game.  It was also my first experience with alt input.

For geometric-optics, I've been adding alt input (and sound) as I go. This feels like a better approach, since I'm now thinking about alt input as part of the original design/implementation, not something later where I might have to make changes. This is also how I'm doing PhET-iO instrumentation these days.
